crates/sui-core/src/consensus_handler.rs

```diff
@@ -970,36 +959,6 @@ mod tests {
         }
     }
 
-    #[test]
-    pub fn test_update_index_and_hash() {
-        let index0 = ExecutionIndices {
-            sub_dag_index: 0,
-            transaction_index: 0,
-            last_committed_round: 0,
-        };
-        let index1 = ExecutionIndices {
-            sub_dag_index: 0,
-            transaction_index: 1,
-            last_committed_round: 0,
-        };
-        let index2 = ExecutionIndices {
-            sub_dag_index: 1,
-            transaction_index: 0,
-            last_committed_round: 0,
-        };
-
-        let mut last_seen = ExecutionIndicesWithStats {
-            index: index1,
-            hash: 1000,
-            stats: ConsensusStats::default(),
-        };
-
-        let tx = &[0];
-        assert!(!update_index_and_hash(&mut last_seen, index0, tx));
-        assert!(!update_index_and_hash(&mut last_seen, index1, tx));
-        assert!(update_index_and_hash(&mut last_seen, index2, tx));
```

Comment:
Not sure if we have other tests asserting the updates made by `update_index_and_hash()`.

Reply:
Good call. It doesn't look like there was other test testing update_index_and_hash. Add a test to check the content now.